src/octoprint/static/js/app/viewmodels/files.js

```diff
@@ -1154,46 +1159,65 @@ $(function () {
-                    ),
-                    {name: _.escape(data.name)}
-                ) +
-                "</p>";
             var info = "";
+            var objectFits = true;
+            var travelFits = true;
 
-            var formatData = {
-                profile: boundaries,
-                object: printingArea
-            };
+            function _area_exceeds_boundaries(ax, area) {
+                return (
+                    area["min" + ax] < boundaries["min" + ax] ||
+                    area["max" + ax] > boundaries["max" + ax]
+                );
+            }
 
-            // find exceeded dimensions
-            if (
-                printingArea["minX"] < boundaries["minX"] ||
-                printingArea["maxX"] > boundaries["maxX"]
-            ) {
+            // check if printing area exceeds boundaries
+            if (_area_exceeds_boundaries("X", printingArea)) {
                 info += gettext("Object exceeds print volume in width.<br>");
+                objectFits = false;
             }
-            if (
-                printingArea["minY"] < boundaries["minY"] ||
-                printingArea["maxY"] > boundaries["maxY"]
-            ) {
+            if (_area_exceeds_boundaries("Y", printingArea)) {
                 info += gettext("Object exceeds print volume in depth.<br>");
+                objectFits = false;
             }
-            if (
-                printingArea["minZ"] < boundaries["minZ"] ||
-                printingArea["maxZ"] > boundaries["maxZ"]
-            ) {
+            if (_area_exceeds_boundaries("Z", printingArea)) {
                 info += gettext("Object exceeds print volume in height.<br>");
+                objectFits = false;
+            }
+
+            // check if travel area exceeds boundaries
+            if (_area_exceeds_boundaries("X", travelArea)) {
+                info += gettext("Travel exceeds print volume in width.<br>");
+                travelFits = false;
+            }
+            if (_area_exceeds_boundaries("Y", travelArea)) {
+                info += gettext("Travel exceeds print volume in depth.<br>");
+                travelFits = false;
+            }
+            if (_area_exceeds_boundaries("Z", travelArea)) {
+                info += gettext("Travel exceeds print volume in height.<br>");
+                travelFits = false;
             }
 
-            //warn user
-            if (info !== "") {
+            if (travelFits && objectFits) {
+                return true;
+            } else {
+                // model not within bounds, we need to prepare a warning
                 if (notify) {
+                    var formatData = {
+                        name: _.escape(data.name),
+                        profile: boundaries,
+                        object: printingArea,
+                        travel: travelArea,
+                        culprit: !objectFits ? "Object" : "Travel area"
```

Comment:
"Object" and "Travel area" need to be made translatable:

```suggestion
                        culprit: !objectFits ? gettext("Object") : gettext("Travel area")
```

Sorry, missed that in my earlier review.

Actually, to make the translation files less repetitive, I'd suggest something like this:

``` js
function _exceed_warning(culprit, dimension) {
    return _.sprintf(gettext("%(culprit)s exceeds print volume in %(dimension)s.<br>"), {culprit: culprit, dimension: dimension});
}
```

and then calling it like this:

``` js
if (_area_exceeds_boundaries("X", printingArea)) {
    info += _exceed_warning(gettext("Object"), gettext("width"));
    objectFits = false;
}

// ...

if (_area_exceeds_boundaries("Z", travelArea)) {
    info += _exceed_warning(gettext("Travel"), gettext("height"));
    travelFits = false;
}
```